.github/workflows/ci.yml

```diff
@@ -51,11 +51,32 @@ jobs:
   test:
     name: Test
     needs: [build]
-    uses: codecov/gha-workflows/.github/workflows/run-tests.yml@v1.2.12
+    uses: codecov/gha-workflows/.github/workflows/run-tests.yml@v1.2.15
     secrets: inherit
     with:
       repo: ${{ vars.CODECOV_IMAGE_V2 || 'codecov/self-hosted-worker' }}
 
+  print:
```

Comment:
Do we need this for prod debugging? Just making sure these weren't left from local testing and are indeed intended

Reply:
it's not intended, thanks for catching that